app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -205,21 +214,13 @@ public void onConnected(Bundle bundle) {
     private void updateChapterPages() {
         final boolean wasOrganizer = mViewPagerAdapter != null
             && mViewPagerAdapter.getCount() == ORGANIZER_PAGES.length;
+        if (organizerCheckCallback != null) {
```

Comment:
Not sure how to do it from within the callback. It is set to null in onPause. (This does not prevent leaking the activity).

I prefer to think of the callback as a nullable, kind of "transient" object. I don't like to create an object to then throw it away (for the reason of clean code).